main.swift

```diff
@@ -57,6 +57,20 @@ extension FileManager {
     }
 }
 
+extension Array {
+    func element(after index: Int) -> Element? {
+        
+        //: handles not only negative indices, but also index == self.count
```

Comment:
How about taking `UInt` as input instead of `Int`, that way you can never have negative indexes?